dist/main.js

```diff
@@ -0,0 +1,147 @@
+const renderer = new Renderer()
+const localStorageManager = new LocalStorageManager()
+
+const getRoster = function () {
+  const $input = $("#team-name-input")
+  if ($input.val().length) {
+    $.get(`/teams/${$input.val()}`, function (data) {
+      $input.val(null)
+      if (data.teammates.length) {
+        renderer.renderPlayers(data)
+      } else {
+        renderer.renderEmptyState(data.teamName)
+      }
+    })
+  } else {
+    renderer.renderAlertedInput($input)
+  }
+}
+
+$("#players-container").on("click", "button.showStats", function () {
+  $player = $(this).closest(".player")
```

Comment:
When declaring a new variable you should declare it with _const_ or _let_, otherwise, the variable will be defined as "_var_"